### PR TITLE
Improved frontend for the Filecoin Paid Retrieval Demo

### PR DESCRIPTION
**Pull request by geoknee**
It is:
* mobile friendly
* dark mode friendly
* does not yet work with micropayments (that we can add back later)
Screenshot 2023-09-14 at 15 13 53
Screenshot 2023-09-15 at 13 10 54

### CODE REVIEW

**Comment by geoknee** [comment]
Still need to wire up the "skip payment " switch, and remove the micropayments one for now.

File: packages/payment-proxy-client/src/App2.tsx
Comment (lalexgap) [inline packages/payment-proxy-client/src/App2.tsx:1]: Could we rename this back to `App.tsx`? That would show the diff on the PR.

File: packages/payment-proxy-client/src/App.tsx [hunk 1]
```diff
@@ -1,58 +1,107 @@
...
+  defaultNitroRPCUrl,
+  fileUrl,
+  hub,
+  initialChannelBalance,
+  provider,
+} from "./constants";
+import { fetchFile } from "./file";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function Copyright(props: any) {
```
Comment (lalexgap) [inline packages/payment-proxy-client/src/App.tsx:43]: Could we type this so we don't have to supress the `any` warning?

File: packages/payment-proxy-client/src/main.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 
+// import "./index.css";
```
Comment (lalexgap) [inline packages/payment-proxy-client/src/main.tsx:4]: Could we remove this?

**Review by lalexgap — APPROVED** [review]

**Pull request merged**